### DISCUSSION TITLE
Error Astroid Menu Options v3.0.4

### DISCUSSION THREAD

**Discussion by vologb** [templaza/astroid-framework#460] [General]
Hi,
I came across an error in v3.0.4 in the Astroid Menu Options.
The input field to specify the width of the MegaMenu is not the width input field. It is in fact the field to specify the text for the BadgeOption. When you change the BadgeText it also changes the Width input field and vice verca. I cannot change the width of the Mega Menu. Maybe you can verify and fix?
Thanks and kind regards
Volker
![Screenshot 2023-11-19 090055](https://github.com/templaza/astroid-framework/assets/151429204/6e570244-4da7-4da9-a1fc-9c17c968c8bc)

**Comment by sonvnn** [comment]
This issue was fixed in 3.0.5. Please update Astroid framework
